carousel.js

```diff
@@ -0,0 +1,80 @@
+const carouselImagewrapper = document.getElementById("carousel__image-wrapper");
+const carouselWrapper = document.getElementById("carousel-wrapper");
+const carouselTitle =document.getElementById("carousel__title-wrapper");
+
+const img1 = {
+    id: "carousel__image",
+    class: "carousel__image",
+    srs: "images/carousel_img_1.png",
+    alt: "eggs and chicken",
+}
+
+const img2 = {
+    id: "carousel__image",
+    class: "carousel__image",
+    srs: "images/carousel_img_2.png",
+    alt: "something more",
+}
+
+const img3 = {
+    id: "carousel__image",
+    class: "carousel__image",
+    srs: "images/carousel_img_3.png",
+    alt: "something more",
+}
+
+const carouselImg = [img1, img2, img3];
+
+for(let i=0;i<carouselImg.length;i++){
+    const img_all = document.createElement("img")
+    img_all.setAttribute("id",carouselImg[i].id);
+    img_all.setAttribute("class", carouselImg[i].class);
+    img_all.setAttribute("src", carouselImg[i].srs);
+    img_all.setAttribute("alt", carouselImg[i].alt);
+    carouselImagewrapper.append(img_all);
+}
+
+var derection=-1;
```

Comment:
If you do `let` instead of `var` will it work?

Reply:
It's works